.mergify.yml

```diff
@@ -4,6 +4,9 @@ pull_request_rules:
       - "#approved-reviews-by>=1"
       - "#review-threads-unresolved=0"
       - "-draft"
+      - "label!=docker"            # Don't auto merge docker images
+      - "#check-failure=0"         # Don't auto merge with a failure
+      - "check-success~=Build"     # Don't auto merge unless a build has succeeded, needed because above is true on a fresh PR before builds
```

Comment:
maybe also `#check-pending=0` to wait for all pending CIs to run.

Reply:
Done